components/root/root.jsx

```diff
@@ -147,7 +149,75 @@ export default class Root extends React.Component {
     }
 
     handleAccessibilityKeys = (e) => {
-        if (isKeyPressed(e, Constants.KeyCodes.TAB)) {
+        if (isKeyPressed(e, Constants.KeyCodes.DOWN)) {
+            const lhsList = document.getElementById('lhsList');
+            if (lhsList.contains(e.target)) {
+                this.sidebarCategories = [
```

Comment:
No need to use `this` as this variable can be local and doesn't need to be stored on the class